.github/workflows/e2e-playwright-submodule.yml

```diff
@@ -0,0 +1,166 @@
+name: CI - E2E - Playwright - Submodule
```

Comment:
Is this necessary? I don't see where it's running in a distinct language...

Reply:
I guess it's just to make sure everything is working with the BUILD_WITH_SUBMODULE env variable. So we can confidently deploy. Once we migrate to the submodule, that variable will be removed along with this action.